src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -197,9 +210,24 @@ internal void AddProvider(string workerDir)
             }
         }
 
-        private List<WorkerDescriptionProfile> ReadWorkerDescriptionProfiles(JToken profilesJToken)
+        private static JsonElement GetWorkerConfigJsonElement(string workerConfigPath)
         {
-            var profiles = profilesJToken.ToObject<IList<WorkerProfileDescriptor>>();
+            ReadOnlySpan<byte> jsonSpan = File.ReadAllBytes(workerConfigPath).AsSpan();
```

Comment:
since you're using `ReadOnlySpan<byte>` here, you can rely on the implicit conversion and drop `AsSpan()`